src/kbmod/search/PointSpreadFunc.cpp

```diff
@@ -11,20 +11,20 @@ namespace search {
 
 PointSpreadFunc::PointSpreadFunc(float stdev) {
     width = stdev;
-    float simpleGauss[MAX_KERNEL_RADIUS];
-    double psfCoverage = 0.0;
-    double normFactor = stdev * sqrt(2.0);
+    float simple_gauss[MAX_KERNEL_RADIUS];
+    double psf_coverage = 0.0;
+    double norm_factor = stdev * sqrt(2.0);
```

Comment:
I'm not sure this jives with the rest of the code I saw - local vars are sometimes camelCase sometimes not? Couple of lines later it's `currentBin` too.

Reply:
Local variables should all be snake case. I missed a bunch in my first pass. But I should have gotten most of them now.